### ISSUE TITLE
installing unstable version using curl returns error

### ISSUE THREAD

**Issue by kacole2** [emccode/rexray#140]
I am trying to install `unstable` using: `curl -sSL https://dl.bintray.com/emccode/rexray/install | sh -s unstable -` but it returns the following error:

```
dpkg-deb: error: `rexray-latest-x86_64.deb' is not a debian format archive
dpkg: error processing archive rexray-latest-x86_64.deb (--install):
 subprocess dpkg-deb --control returned error exit status 2
Errors were encountered while processing:
 rexray-latest-x86_64.deb

REX-Ray has been installed to /usr/bin/rexray

sh: 136: /usr/bin/rexray: not found
```

However, if i install the stable version with `curl -sSL https://dl.bintray.com/emccode/rexray/install | sh -` then no issues

**Comment by clintonskitson** [comment]
@kenny try `curl -sSL https://dl.bintray.com/emccode/rexray/install | sh -s unstable latest -`

**Comment by kacole2** [comment]
still no go. 
```
dpkg-deb: error: `rexray-latest-x86_64.deb' is not a debian format archive
dpkg: error processing archive rexray-latest-x86_64.deb (--install):
 subprocess dpkg-deb --control returned error exit status 2
Errors were encountered while processing:
 rexray-latest-x86_64.deb
```

**Comment by clintonskitson** [comment]
Seeing the same thing @kacole2

**Comment by clintonskitson** [comment]
Looks like it's being caused by missing the `latest` directory when deploying the `unstable` builds.

https://dl.bintray.com/emccode/rexray/unstable/

**Comment by clintonskitson** [comment]
@kenny one step further in investigating.  If you check the Travis-CI build log from RR, you can see that it is failing towards the end.

https://travis-ci.org/emccode/rexray/builds/86234670

It looks like the updated path structure is breaking the upload feature to Bintray.  Will have it fixed shortly.

**Comment by clintonskitson** [comment]
New build going @kenny
https://travis-ci.org/emccode/rexray/builds/86260634

Check your command in a few minutes.

**Comment by clintonskitson** [comment]
```
root@test:/home/ubuntu# curl -sSL https://dl.bintray.com/emccode/rexray/install | sh -s unstable latest -
(Reading database ... 51452 files and directories currently installed.)
Preparing to unpack rexray-latest-x86_64.deb ...
Unpacking rexray (0.2.1+dev+40-1) over (0.2.1+dev+40-1) ...
Setting up rexray (0.2.1+dev+40-1) ...

REX-Ray has been installed to /usr/bin/rexray

Binary: /usr/bin/rexray
SemVer: 0.2.1-dev+40
OsArch: Linux-x86_64
Branch: master
Commit: d1e4513c3d3ec42629b65021da56738863592b68
Formed: Mon, 19 Oct 2015 20:09:02 UTC

root@test:/home/ubuntu# 

```

**Comment by akutz** [comment]
Closed with d1e4513c3d3ec42629b65021da56738863592b68.
